browser/src/canvas/CanvasSectionContainer.ts

```diff
@@ -577,10 +577,30 @@ class CanvasSectionContainer {
 		while (layoutingService.runTheTopTask());
 	}
 
+	private isCanvasSizeValidAfterDisplayChange(): boolean {
+		/*
+			In some situations, we receive resize events that cause the canvas to size to 0x0 and we don't receive a subsequent
+			resize event when the size is restored. As a work-around, if we have a zero-sized canvas,
+			double-check the document container size to see if we've missed a resize event
+		*/
+		if (this.width === 0 || this.height === 0) {
```

Comment:
this is unexpected case - I would put some console warn here so we can catch it later easily